Mypy job failing

The latest, freshest mypy time waster

https://app.circleci.com/pipelines/github/pytorch/vision/17366/workflows/a475b8b1-1463-4159-bf00-be2d82030c0b/jobs/1405650

I think we just need to wait for https://github.com/pytorch/pytorch/pull/76895 to be in the nightlies

```
torchvision/models/_api.py:64: error: Incompatible return value type (got
Module, expected "OrderedDict[Any, Any]")  [return-value]
            return load_state_dict_from_url(self.url, progress=progress)
                   ^
Found 1 error in 1 file (checked 209 source files)

Exited with code exit status 1

CircleCI received exit code 1
```

--- a/torchvision/models/_api.py
+++ b/torchvision/models/_api.py
@@ -1,7 +1,6 @@
 import importlib
 import inspect
 import sys
-from collections import OrderedDict
 from dataclasses import dataclass, fields
 from inspect import signature
 from typing import Any, Callable, Dict, cast
@@ -60,7 +59,7 @@ class WeightsEnum(StrEnum):
                 )
         return obj
 
-    def get_state_dict(self, progress: bool) -> OrderedDict:
+    def get_state_dict(self, progress: bool) -> Dict[str, Any]:
         return load_state_dict_from_url(self.url, progress=progress)
 
     def __repr__(self) -> str:

